### PR TITLE
Add cinema4d docs

### PR DESCRIPTION
**Pull request by MustafaJafar**
## Changelog Description
Add cinema4d docs
TODO: 
- [x] artist docs
- [x] admin docs
- [x] feature card
- [x] badge

## Testing notes:

1. Read through the changes
2. Approve if the changes make sense to you.

### CODE REVIEW

File: website/docs/addon_cinema4d_admin.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+A Qt library must be installed for Cinema4D to ensure the AYON tools can run, either make sure
+a `PySide6` or `PySide2` library is available on `PYTHONPATH` matching the Python version of the Cinema4D release.
+
+Alternatively, you can use the Tray launcher to trigger the `Terminal` tool to open a command-line initialized for the Cinema4D application. From there you should be able to run:
+```cmd
+c4dpy.exe -m pip install --ignore-installed PySide6
+```
+
+TODO: Add info about prelaunch hook `InstallPySideToCinema4D`.
```
Comment (BigRoy) [inline website/docs/addon_cinema4d_admin.md:30]: This is currently in the codebase, but disabled/unused due to issues. No need to document it for now.
```suggestion
```

File: website/docs/addon_cinema4d_artist.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+## Working with AYON in Cinema4D
+
+The AYON Cinema4D integration addon allows you to manage your Cinema4D workfiles, load versioned content and manage it all nicely directly within the AYON ecosystem and easily produce predictable caches back out for easy loading into other integrations in the AYON ecosystem.
+
+## Addon Features
+
+### AYON Menu
+
+Once you launch **Cinema4D** via AYON Launcher, you will be introduced to AYON Menu present in the menu header which is also a clear indication of running **Cinema4D** in AYON context.
```
Comment (BigRoy) [inline website/docs/addon_cinema4d_artist.md:25]:
```suggestion
Once you launch **Cinema4D** via AYON Launcher, an AYON Menu will be present in the menu header which is a clear indication of running **Cinema4D** in AYON context.
```

File: website/docs/addon_cinema4d_artist.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+The AYON Cinema4D integration addon allows you to manage your Cinema4D workfiles, load versioned content and manage it all nicely directly within the AYON ecosystem and easily produce predictable caches back out for easy loading into other integrations in the AYON ecosystem.
+
+## Addon Features
+
+### AYON Menu
+
+Once you launch **Cinema4D** via AYON Launcher, an AYON Menu will be present in the menu header which is a clear indication of running **Cinema4D** in AYON context.
+
+AYON Menu provides access to AYON global tools as well as some workfile actions and pipeline tools.
```
Comment (BigRoy) [inline website/docs/addon_cinema4d_artist.md:27]:
```suggestion
This AYON Menu provides access to AYON's tools for workfiles, loading and managing content and publishing (exporting) from your Cinema4D workfile.
```

File: website/docs/addon_cinema4d_artist.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+### AYON Global Tools
+
+These are the standard AYON core tools in tandem with the Integrations.
+
+### Workfile Actions
+
+They are used to set scene data.
+
+TODO: Add more info about the available workfile actions.
```
Comment (BigRoy) [inline website/docs/addon_cinema4d_artist.md:62]:
```suggestion
- Reset Frame Range: Set the Cinema4D timeline frame range and FPS as defined on the current task in AYON.
- Reset Scene Resolution: Set the render resolution as defined on the current task in AYON.
- Reset Colorspace: Set the OCIO conifg as defined on the current task in AYON.
```

**Review by BigRoy — APPROVED** [review]

**Comment by BigRoy** [comment]
@MustafaJafar can you do one more review pass on this and if all good, feel free to merge.

**Comment by MustafaJafar** [comment]
> @MustafaJafar can you do one more review pass on this and if all good, feel free to merge.

Thank you, it looks great. We may add more visuals later.

**Pull request merged**